assembl/static2/js/app/pages/idea.jsx

```diff
@@ -54,8 +54,35 @@ const noRowsRenderer = () => {
 };
 
 class Idea extends React.Component {
+  topPostsToLoad = (topPosts, edges) => {
+    const { hash } = window.location;
```

Comment:
Again, we're using the framework, which recommends the use its props than window.location. Should use the react-router's prop (passed from parent to its children).

Reply:
@ayyazdaniaryan Ah here for the hash yes, we can retrieve it from react-router's location. It was not the case above it seems for protocol and host. But the react-router do in the end window.location right? I think here we need to add withRouter to have it, so we complexify the code for almost nothing IMO.

Reply:
Hum, I tried to use  `<Idea ... location={props.location} >` in routes.jsx but this.props.location was undefined in getInitialRowIndex... I gave up.